l10n/css.json

```diff
@@ -181,7 +181,7 @@
     "de": "all elements except inline boxes and internal ruby or table boxes",
     "en-US": "all elements except inline boxes and internal ruby or table boxes",
     "fr": "all elements except inline boxes and internal ruby or table boxes",
-    "ja": "インラインボックスおよび内部のルビまたは表ボックスを除くすべての要素",
+    "ja": "all elements except inline boxes and internal ruby or table boxes",
```

Comment:
In every other case, you've removed a `ja` string, added a `ja` string, or modified an existing `ja` string. In this one though, you've replaced the `ja` text with an English equivalent. Did you mean to do this?

Reply:
Sorry, it is my mistake. This PR is based on an older version.
I have recreated new PR on https://github.com/mdn/data/pull/458 instead of this. Please review it instead.

Reply:
No problem at all.